### DISCUSSION TITLE
How to Override

### DISCUSSION THREAD

**Discussion by ppeyman** [breezefront/community#323] [1. Help]
Hi dears i need override Swissup/Breeze/js/components/price-box.js to change some code ? how can i do it ? 
i change in custom theme magento_catalog but doesn't works 
thank you

**Comment by vovayatsyuk** [comment]
Use [mixin](https://breezefront.com/docs/mixins#example) for `priceBox`.

**Reply by ppeyman** [reply]
> Use [mixin](https://breezefront.com/docs/mixins#example) for `priceBox`.

thank you
how can i remove or disable ?
`<item name="priceBox" xsi:type="string">Swissup_Breeze/js/components/price-box</item>`
from breeze_default.xml from Swissup_breeze ? 
i trying in head tag but doesn't removed.
